instrumentation/opentelemetry_phoenix/test/opentelemetry_phoenix_test.exs

```diff
@@ -53,16 +53,16 @@ defmodule OpentelemetryPhoenixTest do
     assert %{
              "http.client_ip": "10.211.55.2",
              "http.flavor": :"1.1",
-             "http.host": "localhost",
+             "net.host.name": "localhost",
```

Comment:
Double-check here, I am not sure if this is correct or not but `http.host` does not exists in the macros

Reply:
Yea, pretty sure this is right.